content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,125 @@
++++
+title = "Fission Functions with ArgoWorkflow"
+date = "2022-04-14T11:35:34+05:30"
+author = "Neha Gupta"
+description = "Create your workflow with Fission and Argo"
+categories = ["Tutorials"]
+type = "blog"
+images = ["images/featured/fission-zapier-featured.png"]
++++
+
+Fission Function allow user to perform one logical task. To group multiple task together such as one function is dependent on other we can use Argo Workflows. Argo Workflows is an open source container-native workflow engine with a feature to create DAGs i.e running task sequentially, parallely and with dependencies. We will try to develop a simple Insurance Eligibility programe which will take different input and calculate insurance instalment on basis of the inputs.
+
```

Comment:
Please check all errors identified by misspell and language tool bots. 
You can mark most of them as resolved which are not relevant.